### PR TITLE
Django 1.9 updates and also clear list

### PR DESCRIPTION
**Pull request by KevinGrahamFoster**
I am calling multiple dump_object calls from the same execution of django, hence the clear list addition.

More than willing to make improvements to this PR before its integrated, please just give me suggestions.

### CODE REVIEW

File: fixture_magic/management/commands/custom_dump.py [hunk 1]
```diff
@@ -7,7 +7,10 @@
     from django.utils import simplejson as json
 
 from django.core.management.base import BaseCommand
-from django.db.models import loading
+try:
+    from django.db.models import loading
+except:
```
Comment (davedash) [inline fixture_magic/management/commands/custom_dump.py]: can you make this more specific, `ImportError` I think is correct.

File: fixture_magic/management/commands/custom_dump.py [hunk 2]
```diff
@@ -7,7 +7,10 @@
...
+    from django.apps import apps
```
Comment (davedash) [inline fixture_magic/management/commands/custom_dump.py]: You could do `... apps as loading`

File: fixture_magic/management/commands/custom_dump.py [hunk 3]
```diff
@@ -23,7 +26,10 @@ def handle(self, dump_name, *pks, **options):
         # Get the primary object
         dump_settings = settings.CUSTOM_DUMPS[dump_name]
         (app_label, model_name) = dump_settings['primary'].split('.')
-        dump_me = loading.get_model(app_label, model_name)
+        try:
+            dump_me = loading.get_model(app_label, model_name)
+        except:
```
Comment (davedash) [inline fixture_magic/management/commands/custom_dump.py]: If you do `apps as loading above` you don't need the `try/except` here.

File: fixture_magic/management/commands/dump_object.py [hunk 4]
```diff
@@ -5,7 +5,10 @@
 from django.core.exceptions import FieldError, ObjectDoesNotExist
 from django.core.management.base import BaseCommand, CommandError
 from django.core.serializers import serialize
-from django.db.models import loading
+try:
+    from django.db.models import loading
+except:
```
Comment (davedash) [inline fixture_magic/management/commands/dump_object.py]: Same comments in this file.

**Comment by davedash** [comment]
Thanks for these changes.  I haven't looked too closely, but if you can make the suggested changes, I'd happily merge this.

I didn't get a chance to run this myself, so hopefully the tests will cover most of this.

**Comment by davedash** [comment]
You might want to change the requirements to `Django>=1.9` or whatever the current version is.  Not 100% necessary for this, but it might make things easier.

Thanks again.

**Comment by KevinGrahamFoster** [comment]
Awesome thank you. I've made the updates you recommended. Let me know if there's anything else I can do. I've been testing this code with my project and django 1.9.2 and all seems well so far.

**Comment by davedash** [comment]
Thanks @KevinGrahamFoster

**Comment by KevinGrahamFoster** [comment]
Thank you! Awesome. :)

**Comment by KevinGrahamFoster** [comment]
@davedash Was wondering when you are planning a release? (And secretly hoping it will be in short order, but no pressure :sake: )

**Comment by davedash** [comment]
Give 0.0.8 a try

**Pull request merged**